code/modules/clothing/head/soft_caps.dm

```diff
@@ -141,6 +141,7 @@
 	bomb = 25
 	fire = 20
 	acid = 50
+	wound = 4
```

Comment:
If they have the same armour why not just use the same datum?

Reply:
yeahh fairr ill push it tommorow